Makefile

```diff
@@ -0,0 +1,27 @@
+name := securebanking-openbanking-uk-rcs
+repo := sbat-gcr-develop
+
+.PHONY: all
+all: clean test package
+
+clean:
+	rm -f ${name}.jar
+	mvn clean
+
+test:
+	mvn verify
+
+package:
+	mvn package -DskipTests -DskipITs --file pom.xml
+
+docker: clean package
+ifndef tag
+	$(error "You must supply a docker tag")
+endif
+	cp ${name}-sample/target/${name}-*.jar ./${name}.jar
+	docker build -t eu.gcr.io/${repo}/securebanking/${name}:${tag} .
+	docker push eu.gcr.io/${repo}/securebanking/${name}:${tag}
+
+dev: clean package
+	cp ${name}-sample/target/${name}-*.jar ./${name}.jar
+	docker build -t eu.gcr.io/${repo}/securebanking/${name}:latest .
```

Comment:
How do we know which version of the jar is being built and pushed? Should we tag with the maven version as well as latest?

Reply:
it isnt being pushed. this target is just for a local build its not used in ci at all. the docker-compose is only looking for the `latest` tag so im only tagging the image with `latest`. The version of the jar file itself is irrelevant